manifest.yml

```diff
@@ -0,0 +1,8 @@
+applications:
+- name: ras-backstage
```

Comment:
Does this need to be `ras-backstage-ui` to avoid conflicts with the Python service?

Reply:
By the looks of it the ui is the "default" one and the python service has the classifier

```
s-backstage-sit                        started           1/1         512M     1G     ras-backstage-sit.apps.devtest.onsclofo.uk
ras-backstage-service-sit                started           1/1         512M     1G     ras-backstage-service-sit.apps.devtest.onsclofo.uk
```